person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id, :name, :age
```

Comment:
- [ ] You currently  do not have `setters` for name & age and as such i would suggest you create them using either the shortcut `attr_accessor` or `attr_writer` :+1: :sparkles: